samples/unit-tests/maps/geojson/demo.js

```diff
@@ -1,8 +1,13 @@
 QUnit.test('GeoJSON and map transforms', assert => {
 
+    // #17139
+    const proj4 = window.proj4;
+    window.proj4 = undefined;
```

Comment:
So now we have a test for the `chart.proj4` option, but we no longer have a test for `window.proj4`. We need to refactor this so both cases are covered. Move the logic to a function, then run the same set of tests. Can probably run `chart.update` in between to change the setting.

Reply:
Sure! In progress